src/CommandingOfficers/Commander.java

```diff
@@ -62,12 +61,15 @@ public void initTurn(GameMap map)
     }
     money += turnIncome;
 
-    // Un-apply any modifiers that were activated last turn.
-    // TODO: If/when we have modifiers that last multiple turns, figure out how to handle them.
-    for( int i = modifiers.size() - 1; i >= 0; --i )
+    // Update any modifiers that were activated last turn, removing expired ones.
+    for( int i = 0; i < modifiers.size(); i++ )
```

Comment:
Why did you remove the iteration direction? If we remove elements while iterating forward, our index will get out of sync.